### PR TITLE
redis cluster now support any region [ch2304]

### PR DESCRIPTION
**Pull request by star0x4b**

### CODE REVIEW

File: tasks/cluster.yml [hunk 1]
```diff
@@ -14,21 +14,24 @@
...
   copy:
     src: redis-trib.rb
     dest: /usr/bin/redis-trib.rb
     mode: 0755
 
 - name: Install redis-discovery from localfile.
-  copy:
-    src: redis-discovery.sh
+  template:
+    src: scripts/redis-discovery.sh.j2
```
Comment (akhon) [inline tasks/cluster.yml]: as for me, would be better to place templates into `templates/` folder

Comment (star0x4b) [reply tasks/cluster.yml]: this path is relative to templates/ folder

Comment (akhon) [reply tasks/cluster.yml]: 👍

Comment (max-lobur) [reply tasks/cluster.yml]: I'd put to `./templates/redis-cluster/redis-discovery.sh.j2` tho
trying to keep templates dir split by roles, or at least be able to guess which roll is that

Comment (star0x4b) [reply tasks/cluster.yml]: again, this path is relative to templates/ folder ~inside~ redis-cluster role. It should be easy to guess to which role this template belongs :)

Comment (max-lobur) [reply tasks/cluster.yml]: ah, I see, nvm then

**Pull request merged**